### PR TITLE
feat(condo): Doma-9014 add online interaction with billing integration

### PR DESCRIPTION
**Pull request by dkoviazin**

### CODE REVIEW

File: apps/condo/domains/billing/utils/serverSchema/checkBillingAccountNumberForOrganization.js [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+const { CONTEXT_FINISHED_STATUS } = require('@condo/domains/acquiring/constants/context')
+const { AcquiringIntegrationContext } = require('@condo/domains/acquiring/utils/serverSchema')
+const { BillingIntegrationOrganizationContext, BillingAccount } = require('@condo/domains/billing/utils/serverSchema')
+
+const ONLINE_INTERACTION_CHECK_ACCOUNT_SUCCESS_STATUS = 'FOUND'
+const ONLINE_INTERACTION_CHECK_ACCOUNT_ERROR_STATUS = 'ERROR'
+const ONLINE_INTERACTION_CHECK_ACCOUNT_NOT_FOUND_STATUS = 'NOT_FOUND'
+
+const appLogger = getLogger('condo')
+const logger = appLogger.child({ module: 'condo-billing-online-interaction' })
```
Comment (pahaz) [inline apps/condo/domains/billing/utils/serverSchema/checkBillingAccountNumberForOrganization.js:16]: Why not just getLogger?

File: apps/condo/domains/billing/utils/serverSchema/checkBillingAccountNumberForOrganization.js [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+
+const checkAccountNumberWithOnlineInteractionUrl = async (interactionUrl, tin, accountNumber) => {
+    const url = new URL(interactionUrl)
+    url.searchParams.append('tin', tin)
+    url.searchParams.append('accountNumber', accountNumber)
+    try {
```
Comment (pahaz) [inline apps/condo/domains/billing/utils/serverSchema/checkBillingAccountNumberForOrganization.js:22]: Do we need to check domains?

File: apps/condo/domains/resident/schema/RegisterServiceConsumer.spec.js [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+    MeterTestMixin,
+} = require('@condo/domains/billing/utils/testSchema/testUtils')
+const { ERRORS: { BILLING_ACCOUNT_NOT_FOUND, ACCOUNT_NUMBER_IS_NOT_SPECIFIED } } = require('@condo/domains/resident/schema/RegisterServiceConsumerService')
+const { registerServiceConsumerByTestClient } = require('@condo/domains/resident/utils/testSchema')
+
+
+
+describe('BillingIntegration with online interaction', () => {
+
+    // Проверить в раздельном потоке
```
Comment (pahaz) [inline apps/condo/domains/resident/schema/RegisterServiceConsumer.spec.js:27]: !

File: apps/condo/domains/resident/schema/ServiceConsumer.js [hunk 4]
```diff
@@ -100,6 +100,7 @@ const ServiceConsumer = new GQLListSchema('ServiceConsumer', {
         // todo(@toplenboren) DOMA-1701 Make this deprecated and add mobile an ability to move away from these fields
         // The reason for this field is to avoid adding check for resident user into global Organization read access.
         // This field have specific use case for mobile client.
+        // [DEPRECATED]
```
Comment (pahaz) [inline apps/condo/domains/resident/schema/ServiceConsumer.js:103]: It’s better to add it inside schema doc

Comment (pahaz) [reply apps/condo/domains/resident/schema/ServiceConsumer.js:103]: https://www.apollographql.com/tutorials/lift-off-part5/06-field-deprecation

File: apps/condo/jest.mocksSetup.js [hunk 5]
```diff
@@ -1,4 +1,5 @@
 // The mocked module implementation must be placed into __mocks__/MockedClass near the module itself
 console.log('🥸Mock some modules')
+jest.mock('@open-condo/keystone/fetch')
```
Comment (SavelevMatthew) [inline apps/condo/jest.mocksSetup.js:3]: ?

We decided not to use this

**Review by SavelevMatthew — CHANGES_REQUESTED** [review]
Why do you mock fetch?

**Pull request closed without merging**